pkg/kubelet/config/file_linux.go

```diff
@@ -151,3 +154,28 @@ func (s *sourceFile) consumeWatchEvent(e *watchEvent) error {
 	}
 	return nil
 }
+
+func modifyStaticPodForPinnedManagement(pod *v1.Pod) {
+	if pod.ObjectMeta.Annotations == nil {
+		pod.ObjectMeta.Annotations = make(map[string]string)
+	}
+	pod.ObjectMeta.Annotations["io.openshift.management"] = "true"
```

Comment:
we opted for a single annotation here:
`io.openshift.management`
if it is set, but it's value is empty, no cpushares are configured, otherwise, they're configured

though, because it is ctr level anyway, this distinction may not even really be possible...

Reply:
apparently it was decided we aren't doing a single annotation anyway

Reply:
removed the annotation, but kept it in the test as expected.